dev/tasks/tasks.yml

```diff
@@ -815,7 +815,10 @@ tasks:
     ci: github
     template: docker-tests/github.linux.yml
     params:
-      flags: -e ARROW_USE_MESON=ON
+      # ARROW_USE_CACHE=OFF is for using sccache
```

Comment:
Does this mean ccache won't be used here? CMake-based docker builds use ccache by default, which is very useful for local development work.

Reply:
Yes - instead of using ccache this will use sccache.

Reply:
Note that this is from the request in https://github.com/apache/arrow/pull/45858#issuecomment-2741723075